cocos/2d/CCFontAtlas.cpp

```diff
@@ -353,7 +353,7 @@ bool FontAtlas::prepareLetterDefinitions(const std::u16string& utf16Text)
             tempDef.validDefinition = true;
             tempDef.width = tempRect.size.width + _letterPadding + _letterEdgeExtend;
             tempDef.height = tempRect.size.height + _letterPadding + _letterEdgeExtend;
-            tempDef.offsetX = tempRect.origin.x + adjustForDistanceMap + adjustForExtend;
+            tempDef.offsetX = tempRect.origin.x + adjustForDistanceMap - adjustForExtend;
```

Comment:
Should `adjustForDistanceMap` also be negative like:
``c++
tempDef.offsetX = tempRect.origin.x - adjustForDistanceMap - adjustForExtend;
```
BTW, how did you find this issue? Any test case for it?